servicetalk-loadbalancer-experimental-provider/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+  implementation platform(project(":servicetalk-dependencies"))
+  testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
+
+  api project(":servicetalk-client-api")
+  api project(":servicetalk-concurrent-api")
```

Comment:
The only `api` dependency should be `:servicetalk-http-api`. Others are transitive dependency through `http-api`. If you'd like, you can keep an explicit definition of `:servicetalk-client-api` as well, but there is no dependency on `:servicetalk-concurrent-api`.